tempesta_fw/sched/tfw_sched_http.c

```diff
@@ -1,7 +1,78 @@
 /**
  *		Tempesta FW
  *
+ * Tempesta HTTP scheduler (load-balancing module).
```

Comment:
The changes in the scheduler have nothing in common with configuration framework and must be splitted to separate patch.